### PR TITLE
{chem}[intel/2020a] ABINIT v9.2.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/0b1297c1d0aa768b93522dc123614af3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2627.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/3920f1d86f9036fd3566e4fe61282800 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--parallel 1"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2602.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/d811e07144bb4ed62d87f53b66a11510 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/f3472cf6bd51bd05339221e2069076bd for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0206u40a.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/adbddfc9b9f66d18d22bd328cdb457a4 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
node356.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/169a17b9933849c1642e4c62db19a005 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/84c22b661084c699bb178cdb576e5094 for a full test report.

File: easybuild/easyconfigs/a/ABINIT/ABINIT-9.2.1-intel-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#
```
Comment (lexming) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-9.2.1-intel-2020a.eb:1]: Minor fix
```suggestion
```

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/ABINIT/ABINIT-9.2.1-intel-2020a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/a/ABINIT/ABINIT-9.2.1-intel-2020a.eb]: @boegel I'll apply the minor fix

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**